dandiapi/zarr/tests/test_zarr_upload.py

```diff
@@ -33,8 +40,11 @@ def test_zarr_rest_upload_start(
     )
     assert resp.status_code == 200
     assert resp.json() == [HTTP_URL_RE]
-    if isinstance(storage, S3Storage):
+
+    if embargoed and isinstance(storage, S3Storage):
         assert 'x-amz-tagging' in resp.json()[0]
+    else:
+        assert 'x-amz-tagging' not in resp.json()[0]
```

Comment:
If we're going to make this stronger assertion about things being absent, I think we should test for the absence of `"embargoed"` instead.

While I expect that `"x-amz-tagging"` wouldn't be included in a non-embargoed case, it isn't essential for correctness. In other words, if `"x-amz-tagging"` starts appearing for some other reason, this test shouldn't fail.

For symmetry, maybe look for the presence of `"embargoed"` in the embargoed case too.

Reply:
I mean test for the string `"embargoed=true"` (which is the actual tagged value), rather than test for the string `"x-amz-tagging"`.

Saying "this URL shouldn't be setting tags for embargoing" is what we actually want to assert, rather "this URL shouldn't be setting any tags".

If the URL happens to include `"x-amz-tagging"` (as it did when we were passing the empty string before), that alone isn't really a problem.

Reply:
Oh, I completely agree. However, the tag itself isn't actually included in the pre-signed URL, as far as I can tell. In any pre-signed URL that I've created, it never includes `embargoed=true` or even just `embargoed` anywhere in the string. It just includes `x-amz-tagging`. This appears to also be the case with other headers as well, like `content-md5` and `x-amc-acl`.

I probably should have explained that searching for `x-amz-tagging` was done out of necessity. I would much rather search for just `embargoed=true`.

Reply:
Ah, I see, sorry for my confusion. This is fine then.